base/setup/reactos/resource.h

```diff
@@ -34,11 +34,13 @@
 #define IDC_KEYBOARD         2033
 #define IDC_KEYLAYOUT        2034
 
-#define IDD_DRIVEPAGE        2040
-#define IDC_PARTITION        2041
-#define IDC_PARTCREATE       2042
-#define IDC_PARTDELETE       2043
-#define IDC_DEVICEDRIVER     2044
+#define IDD_DRIVEPAGE           2040
+#define IDC_PARTITION           2041
+#define IDC_INITDISK            2042
+#define IDC_PARTCREATE          2043
+#define IDC_PARTDELETE          2044
+#define IDC_DEVICEDRIVER        2045
+#define IDC_PARTMOREOPTS        2046
```

Comment:
superfluous and counterproductive change
```suggestion
#define IDD_DRIVEPAGE        2040
#define IDC_PARTITION        2041
#define IDC_INITDISK         2042
#define IDC_PARTCREATE       2043
#define IDC_PARTDELETE       2044
#define IDC_DEVICEDRIVER     2045
#define IDC_PARTMOREOPTS     2046
```